magentov2/plugin/Indodana/PayLater/Helper/Transaction.php

```diff
@@ -0,0 +1,514 @@
+<?php
+
+namespace Indodana\PayLater\Helper;
+use IndodanaCommon\IndodanaInterface;
+use IndodanaCommon\IndodanaCommon;
+use IndodanaCommon\IndodanaConstant;
+use Magento\Framework\App\Helper\AbstractHelper;
+use \Magento\Framework\App\Filesystem\DirectoryList;
+
+class Transaction extends AbstractHelper implements IndodanaInterface
+{
+  private $indodanaCommon;
+  protected $_helper;
+  protected $_customer;
+  protected $_urlInterface;
+  protected $_dir;
+  public function __construct(
```

Comment:
For all function, please add newline before and after the function.

Example: 
```
protected $_dir;

public function __construct(
 ...
)
{
  ...
}

public function getIndodanaCommon()
...
```

Please apply this on all changes on this PR

Reply:
Also for all code block, please use brackets `{}` on new line.

Instead of:
```
public function __construct(
  ...
) {
  ...
}
```
This is the correct one:
```
public function __construct(
  ...
)
{ -> newline
  ...
}
```

Please apply this on all changes on this PR